### PR TITLE
💄#286: translate recaptcha

### PR DESCRIPTION
**Pull request by xusandrew**
## Description of Changes

Adds translations for the recaptcha box

Note -- the only way I was able to simulate the recaptcha failing was to let it time out (60 sec), and it correctly displayed the error in the right language.

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [x] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files

### CODE REVIEW

**Review by anncatton — COMMENTED** [review]
Thanks for adding this! just a smol suggested change

**Review by samrichca — APPROVED** [review]
great work!

File: apps/consent-ui/src/components/common/Form/RecaptchaCheckbox/index.tsx [hunk 1]
```diff
@@ -24,18 +24,26 @@ import ReCAPTCHA from 'react-google-recaptcha';
...
 }) => {
 	const { RECAPTCHA_SITE_KEY } = useAppConfigContext();
 
 	return RECAPTCHA_SITE_KEY ? (
-		<ReCAPTCHA ref={recaptchaCheckboxRef} sitekey={RECAPTCHA_SITE_KEY} onChange={onChange} />
+		<ReCAPTCHA
+			ref={recaptchaCheckboxRef}
+			sitekey={RECAPTCHA_SITE_KEY}
+			onChange={onChange}
+			hl={currentLang === 'fr' ? 'fr-CA' : currentLang}
```
[the remark this answers is not in the document]
Comment (samrichca) [reply apps/consent-ui/src/components/common/Form/RecaptchaCheckbox/index.tsx]: I thought this feedback got implemented already. I agree, please update to use enums and `en`

File: apps/consent-ui/src/components/common/Form/RecaptchaCheckbox/index.tsx [hunk 2]
```diff
@@ -24,18 +24,26 @@ import ReCAPTCHA from 'react-google-recaptcha';
...
 }) => {
 	const { RECAPTCHA_SITE_KEY } = useAppConfigContext();
 
 	return RECAPTCHA_SITE_KEY ? (
-		<ReCAPTCHA ref={recaptchaCheckboxRef} sitekey={RECAPTCHA_SITE_KEY} onChange={onChange} />
+		<ReCAPTCHA
+			ref={recaptchaCheckboxRef}
+			sitekey={RECAPTCHA_SITE_KEY}
+			onChange={onChange}
+			hl={currentLang === ValidLanguage.Enum.fr ? 'fr-CA' : 'en'}
```
Comment (anncatton) [inline apps/consent-ui/src/components/common/Form/RecaptchaCheckbox/index.tsx:45]: it looks like `.Enum` and `.enum` are the same according to intellisense, but we've been using `enum` so let's stick with that convention for consistency

**Review by anncatton — APPROVED** [review]
Thanks @xusandrew !

**Pull request merged**